### ISSUE TITLE
Add A2C implementation

### ISSUE THREAD

**Issue by RobertTLange** [RobertTLange/gymnax-blines#2]
Reminder todo after internship.

Mostly for meta-bandit and gridworld tasks

**Comment by DavidSlayback** [comment]
Suggestion: you could probably just implement it as PPO with fixed parameters (gae=1, no advantage normalization, 1 epoch, 1 minibatch, no value clipping) as per "[A2C is a Special Case of PPO](https://arxiv.org/abs/2205.09123)"

**Comment by RobertTLange** [comment]
Good point, I didn't know about this equivalence. For the meta-RL setups I may have to write some extra logic but will try to keep things minimal.
